### ISSUE TITLE
[20220812] test_issue_imagej_hyperstack_arg and test_issue_description_overwrite fail

### ISSUE THREAD

**Issue by olebole** [cgohlke/tifffile#148]
When trying to package version 20220812 for Debian, I see the following failures when running on the installed package:
```
_______________________ test_issue_imagej_hyperstack_arg _______________________

    def test_issue_imagej_hyperstack_arg():
        """Test write ImageJ with hyperstack argument."""
        # https://stackoverflow.com/questions/73279086
        with TempFileName('imagej_hyperstack_arg') as fname:
            data = numpy.zeros((4, 3, 10, 11), dtype=numpy.uint8)
            imwrite(
                fname,
                data,
                imagej=True,
                metadata={'hyperstack': True, 'axes': 'TZYX'},
            )
>       with TiffFile(fname) as tif:

test_tifffile.py:2246: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/lib/python3/dist-packages/tifffile/tifffile.py:3781: in __init__
    fh = FileHandle(file, mode=mode, name=name, offset=offset, size=size)
/usr/lib/python3/dist-packages/tifffile/tifffile.py:13023: in __init__
    self.open()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <tifffile.FileHandle 'test_imagej_hyperstack_arg.tif'>

    def open(self) -> None:
        """Open or re-open file."""
        if self._fh is not None:
            return  # file is open
    
        if isinstance(self._file, os.PathLike):
            self._file = os.fspath(self._file)
    
        if isinstance(self._file, str):
            # file name
            self._file = os.path.realpath(self._file)
            self._dir, self._name = os.path.split(self._file)
>           self._fh = open(self._file, self._mode)  # type: ignore
E           FileNotFoundError: [Errno 2] No such file or directory: '/tmp/test_imagej_hyperstack_arg.tif'

/usr/lib/python3/dist-packages/tifffile/tifffile.py:13038: FileNotFoundError
```
The same for `test_issue_description_overwrite`. The test command is
```shell
cd tests; python3 -m pytest -v
```
Test config:
* platform linux -- Python 3.10.5, pytest-7.1.2, pluggy-1.0.0+repack -- /usr/bin/python3
* versions (Installing the packages marked with "N/A" doesn't change the picture): 
  - tifffile-2022.8.12, 
  - imagecodecs-N/A, 
  - numpy-1.21.5, 
  - zarr-N/A,
  -  dask-N/A,
  -  xarray-N/A,
  -  fsspec-2022.5.0
* test config: SKIP_CODECS | SKIP_DASK | SKIP_LARGE | SKIP_PRIVATE | SKIP_PUBLIC | SKIP_VALIDATE | SKIP_WIN | SKIP_ZARR

Full test log [here](https://ci.debian.net/data/autopkgtest/testing/amd64/t/tifffile/24739552/log.gz). I see the same result when running with the version directly installed from PyPI.

This didn't happen with version 20220803.

**Comment by cgohlke** [comment]
Thank you. It's an indentation error in the tests. I did not notice probably because the temp file wasn't deleted fast enough before it was re-opened...

**Comment by olebole** [comment]
I can confirm this. With [this patch](https://salsa.debian.org/python-team/packages/tifffile/-/raw/3c5ee47d28fed22cfa18f2ff8c067c3dd8d18482/debian/patches/Fix-indentation-error-in-test.patch), it works fine.

**Comment by cgohlke** [comment]
Fixed in v2022.10.10.
